Assets/Scripts/BowlingBall.cs

```diff
@@ -1,151 +0,0 @@
-﻿using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.Security.AccessControl;
-using UnityEngine;
-using UnityEngine.EventSystems;
-using UnityEngine.Experimental.UIElements;
-using UnityEngine.XR.WSA;
-
-public class BowlingBall : MonoBehaviour
-{
-
-	// Use this for initialization
-	public Vector3 InitialVelocity = Vector3.forward;
-	public Vector3 InitialPosition;
-	public float TranslationCoefficient;
-
-	private bool isMoving;
-	private Rigidbody ballRigidbody;
-	private AudioSource audioSource;
-	private string translationDirection;
-	private const float LANE_BOUNDS = 0.45f;
-	private Vector3 GUTTERBALL_BOUNDS = new Vector3(1, -1, 27);
-	private bool readyToLaunch;
-	private bool hasLaunched;
-	private bool ballEnteredBox;
-
-
-
-
-	void Start()
-	{
-		ballRigidbody = GetComponent<Rigidbody>();
-		audioSource = GetComponent<AudioSource>();
-		ballRigidbody.useGravity = false;
-	}
-
-	private void Update()
-	{
-
-		if (translationDirection == "right")
-		{
-			TranslateBall(Vector3.right);
-		}
-		else if (translationDirection == "left")
-		{
-			TranslateBall(Vector3.left);
-		}
-	}
-
-	
-
-	public void Launch(Vector3 launchVelocity)
-	{
-		if (ballRigidbody != null && !isMoving && readyToLaunch)
-		{
-			ballRigidbody.useGravity = true;
-			ballRigidbody.velocity = launchVelocity;
-			audioSource.Play();
-			isMoving = true;
-			readyToLaunch = false;
-			hasLaunched = true;
-		}
-	}
-	
-	public bool HasLaunched()
-	{
-		return hasLaunched;
-	}
-	
-	public void DebugLaunch()
-	{
-		Launch(Vector3.forward * 8f);	
-	}
-
-	public void SetReadyToLaunch( )
-	{
-		readyToLaunch = true;
-		hasLaunched = false;
-	}
-
-	public bool ReadyToLaunch()
-	{
-		return readyToLaunch;
-	}
-
-	public bool IsMoving()
-	{
-		if (ballRigidbody.velocity == Vector3.zero)
-		{
-			isMoving = false;
-		}
-
-		return isMoving;
-	}
-
-	public bool IsOffLane()
-	{
-		return (transform.position.x > GUTTERBALL_BOUNDS.x ||
-		        transform.position.x < -GUTTERBALL_BOUNDS.x ||
-		        transform.position.z > GUTTERBALL_BOUNDS.z ||
-		        transform.position.y < GUTTERBALL_BOUNDS.y);
```

Comment:
Fine for here, but a more designer friendly approach might have been to define the GUTTERBALL_BOUNDS using another object with a BoxCollider or a custom editor widget - so that it could be placed in the UnityEditor.